main/src/com/google/refine/util/LocaleUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.google.refine.util;
+
+import java.util.Locale;
+import java.util.Map;
+
+public class LocaleUtils {
+    public static void setLocale(String code) {
+        if (code == null) {
+            return;
+        }
+        Map<String, String> languageCodeMap = Map.of(
```

Comment:
It could be worth explaining why this translation is needed (what are the language code schemes used on both sides, how to find out if more mappings should be added). By the way, it could be worth mapping for all the cases where the language codes differ in those schemes, even if those languages are not supported by OpenRefine yet, just to be on the safe side (we will probably not remember to edit this when new languages are added).

Reply:
The language code scheme for java's locale system follows [ISO 639-2](https://www.loc.gov/standards/iso639-2/php/code_list.php).

I tried accessing the docs to retrieve that for weblate but noticed that there was some issue with the [docs website](https://docs.openrefine.org/)

Reply:
Sorry about that, I just fixed the docs, it should be up in a few minutes.